tiered-storage/jcloud/src/main/java/org/apache/bookkeeper/mledger/offload/jcloud/provider/JCloudBlobStoreProvider.java

```diff
@@ -162,6 +165,28 @@ public void buildCredentials(TieredStorageConfiguration config) {
         }
     },
 
+
+    /**
+     * Aliyun OSS is compatible with the S3 API
+     * https://www.alibabacloud.com/help/doc-detail/64919.htm
+     */
+    ALIYUN_OSS("aliyun-oss", new AnonymousProviderMetadata(new S3ApiMetadata(), "")) {
+        @Override
+        public void validate(TieredStorageConfiguration config) throws IllegalArgumentException {
+            VALIDATION.validate(config);
+        }
+
+        @Override
+        public BlobStore getBlobStore(TieredStorageConfiguration config) {
+            return OSS_BLOB_STORE_BUILDER.getBlobStore(config);
```

Comment:
`S3_STANDARD_BLOB_STORE_BUILDER` or other names? It seems that `oss` is a universal name.

Reply:
if use `OSS_BLOB_STORE_BUILDER`,`OSS_VALIDATION` , it will only belongs to OSS.
But using `S3_API_BLOB_STORE_BUILDER`,`S3_API_VALIDATION` make it universal, both make sense.
What's your suggestion

Reply:
`AWS_S3` enum is already compatible with S3 API.
If this code is exclusive to Aliyun oss, maybe we don't need to extract a constant?